app/views/pg_hero/home/data_distribution.html.erb

```diff
@@ -0,0 +1,59 @@
+<div class="content">
+  <h1>Data Distribution</h1>
+  <h3>Shard Colocation Groups</h3>
+  <p><b><span style="color: #09436a;">NOTE: </span></b>The colocations include only shards of the tables that are <em>hash</em> distributed.</p>
+  <table class="table colocations-table">
+    <thead>
+      <tr>
+        <th style="width: 10%;"><%= link_to "Node ID", {} %></th>
+        <th style="width: 22%;"><%= link_to "Coloc. Group Size", {sort: "colocated_shards_size"} %></th>
+        <th colspan = "3">Shard Names</th>
+        <th>Shard Sizes</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @colocated_shard_sizes.each do |colocated_shards| %>
+        <% shard_names = colocated_shards[:shard_group].delete('{}').split(',') %>
+        <% shard_sizes = colocated_shards[:each_shard_size].delete('{}').split(',') %>
+        <% shards = shard_names.zip(shard_sizes).to_h %>
+        <% shards = shards.sort.to_h %>
+        <tr>
+          <td rowspan = "<%= shards.length %>" style="width: 10%;">
+            <span style="word-break: break-all;">
+              <%= number_with_delimiter(colocated_shards[:nodeid]) %>
+            </span>
+          </td>
+          <td rowspan = "<%= shards.length %>" style="width: 22%;">
+            <span style="word-break: break-all;">
+              <%= PgHero.pretty_size(colocated_shards[:colocated_shards_size]) %>
+            </span>
+          </td>
+          <td colspan = "3">
+            <span style="word-break: break-all;">
+              <%= shards.keys.first %>
+            </span>
+          </td>
+          <td>
+            <span style="word-break: break-all;">
+              <%= PgHero.pretty_size(shards.values.first) %>
+            </span>
+          </td>
+        </tr>
+        <% shards.drop(1).each do |name, size| %>
```

Comment:
what is the purpose of drop(1)?

Reply:
Could you please change it like this;

```html
<table>
  <tr>
    <td>colum 1</td>
    <td>colum 2</td>
    <td>
      <table>
        <tr>
          <td>shards column 1</td>
          <td>shards colum 2</td>
        </tr>
      </table>
    </td>
</table>
```

Reply:
I'm not sure how a table within a table row would look like. 🤔

Reply:
It should be fine, it is common practice. You might need to add some CSS though to make it look nice.